mac/mac.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "libmac",
+  "buildsystem": "simple",
+  "build-commands": [
+    "make -f Source/Projects/NonWindows/Makefile prefix=/app all install"
```

Comment:
You can probably just use `make-install-args` and `no-autogen: true`.

Reply:
I'm just the messenger in this case: I just copy-paste the Fre:ac implementation every time, and this shared-module is supposed to streamline that a bit.